src/towncrier/test/test_build.py

```diff
@@ -405,6 +417,7 @@ def test_draft_no_date(self):
             call(["git", "init"])
             call(["git", "config", "user.name", "user"])
             call(["git", "config", "user.email", "user@example.com"])
+            call(["git", "config", "commit.gpgSign", "false"])
```

Comment:
We might need to refactor this code into an helper function that does the git init for us.

I expect that this is needed in other tests.

We can leave this as it is for now.

Just a nice to have thing.

Reply:
How about this?